float32 x
float32 y
Image data
